boards/arm/nucleo_g474re/nucleo_g474re.dts

```diff
@@ -173,6 +173,12 @@
 	status = "okay";
 };
 
+&dac1 {
+	pinctrl-0 = <&dac1_out1_pa4>;
```

Comment:
The pin PA4 should also be mentioned in the docs (for all new boards in this PR).

Reply:
dac1_out1_pa4 added to the doc of the nucleo_g474re